content/docs/guides/benchmarking-latency.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Benchmarking latency in Neon's serverless Postgres
+subtitle: Techniques for obtaining meaningful latency data in serverless database environments
+enableTableOfContents: true
+updatedOn: '2025-05-21T00:00:00.000Z'
+---
+
+Benchmarking latency in Neon's serverless Postgres environment presents unique challenges due to compute auto-suspension, connection protocol differences, and geographic distribution. This guide provides detailed methodologies for separating cold-start costs from operational latency, selecting optimal connection types, and designing tests that accurately reflect production conditions.
```

Comment:
Maybe we could add something like "benchmarking is always hard to get right. Benchmarking query latencies is also a very complicated topic. And Neon's serverless platforms adds additional caveats on top" - basically highlighting that it's not just because of Neon that this is challenging.